open-banking-accelerator/accelerators/ob-bi/README.md

```diff
@@ -0,0 +1,16 @@
+### **WSO2 Open Banking BI Accelerator 3.0.0**
+
+**Try Locally:**
+
+Prerequisites:
+1. Download WSO2 SI product 
```

Comment:
Better to rename as Stream Integrator. And if possible update all  ob-apim,ob-is and ob-bi readme files with which  base product versions are supporting..

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/e19df9383f6ae2b74becba71cc2e1341a9a489bc